### PR TITLE
feat(archwiki): init

### PR DESCRIPTION
**Pull request by GenShibe**
## 🎉 Theme for [Arch WIki](https://wiki.archlinux.org/) 🎉
Arch wiki is the wiki page for all things arch/arch related

## 💬 Additional Comments 💬
Despite there being a toggle, don't use the built in dark mode

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

File: styles/archwiki/Catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent]; 
+    
+    // Start styling your website here:
+    #archnavbar {
+        background-color: @mantle !important;
+          border-bottom: 5px @blue solid !important;
```
Comment (uncenter) [inline styles/archwiki/Catppuccin.user.css:172]:
```suggestion
          border-bottom-color: @blue;
```

File: styles/archwiki/Catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+    }
+    
+    .mw-body{
+        background-color: @base !important;
+    }
+    
+    .archwiki-template-meta-related-articles p {
+        background-color: @surface0;
+            color: @text;
+          border-bottom: 5px @blue solid
```
Comment (uncenter) [inline styles/archwiki/Catppuccin.user.css:220]:
```suggestion
          border-bottom-color: @blue;
```

File: scripts/userstyles.yml [hunk 3]
```diff
@@ -123,6 +123,13 @@ userstyles:
         > [!NOTE]
         > This theme applies to [AniChart](https://anichart.net/) as well, which is an extension of AniList.
       current-maintainers: [*anubisnekhet]
+  arch-wiki:
+      name: Arch Wiki
+      category: productivity
+      readme:
+      app-link: "https://wiki.archlinux.org/"
+       usage: "Make sure to use the default **Light** theme"
+      current-maintainers: [*genshibe]
```
Comment (uncenter) [inline scripts/userstyles.yml:132]:
```suggestion
    name: Arch Wiki
    category: productivity
    readme:
    app-link: "https://wiki.archlinux.org/"
      usage: "Make sure to use the default **Light** theme"
    current-maintainers: [*genshibe]
```

File: scripts/userstyles.yml [hunk 4]
```diff
@@ -123,6 +123,13 @@ userstyles:
         > [!NOTE]
         > This theme applies to [AniChart](https://anichart.net/) as well, which is an extension of AniList.
       current-maintainers: [*anubisnekhet]
+  arch-wiki:
+    name: Arch Wiki
+    category: productivity
+    readme:
+    app-link: "https://wiki.archlinux.org/"
```
Comment (uncenter) [inline scripts/userstyles.yml:130]:
```suggestion
      app-link: "https://wiki.archlinux.org/"
```

File: scripts/userstyles.yml [hunk 5]
```diff
@@ -123,6 +123,13 @@ userstyles:
         > [!NOTE]
         > This theme applies to [AniChart](https://anichart.net/) as well, which is an extension of AniList.
       current-maintainers: [*anubisnekhet]
+  arch-wiki:
+    name: Arch Wiki
+    category: productivity
+    readme:
+      app-link: "https://wiki.archlinux.org/"
+      usage: "Make sure to use the default **Light** theme"
+    current-maintainers: [*genshibe]
```
Comment (uncenter) [inline scripts/userstyles.yml:132]:
```suggestion
      current-maintainers: [*genshibe]
```

File: styles/arch-wiki/Catppuccin.user.css
Comment (isabelroses) [inline styles/arch-wiki/Catppuccin.user.css:1]: can you remove this file

File: catwalk.webp
Comment (isabelroses) [inline catwalk.webp:1]: can you remove this file

**Review by isabelroses — CHANGES_REQUESTED** [review]

**Review by isabelroses — APPROVED** [review]

**Pull request merged**